runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -168,8 +171,27 @@ public Token appendCheckpoint(Token snapshotTimestamp, Optional<LivenessUpdater>
                 .begin();
 
         log.info("appendCheckpoint: Started checkpoint for {} at snapshot {}", streamId, snapshotTimestamp);
+        if (LogReplicationUtils.skipCheckpointFor(streamId)) {
+           livenessUpdater.ifPresent(LivenessUpdater::notifyOnSyncComplete);
+           try { // Certain tables are ephemeral and do not need to be checkpointed.
+               startCheckpoint(snapshotTimestamp); // We also do not want to throw trimmed exceptions.
+               finishCheckpoint(); // To achieve this we append an empty checkpoint here.
+               StreamAddressSpace streamAddressSpace = rt.getSequencerView()
+                       .getStreamAddressSpace(new StreamAddressRange(streamId, Address.MAX, Address.NON_ADDRESS));
+               long addressCount = streamAddressSpace.size();
```

Comment:
addressCount will be 0 as checkpoint has run.  Is there any value in logging it?

Reply:
i thought it will just help validate that no entries have been synced?